src/main/java/frc/robot/Constants.java

```diff
@@ -30,9 +40,72 @@ public static class Vision {
     // The layout of the AprilTags on the field
     public static final AprilTagFieldLayout kTagLayout = AprilTagFields.kDefaultField.loadAprilTagLayoutField();
 
-    // The standard deviations of our vision estimated poses, which affect correction rate
-    // (Fake values. Experiment and determine estimation noise on an actual robot.)
+    // The standard deviations of our vision estimated poses, which affect correction rate (Fake values. Experiment and
+    // determine estimation noise on an actual robot.)
     public static final Matrix<N3, N1> kSingleTagStdDevs = VecBuilder.fill(4, 4, 8);
     public static final Matrix<N3, N1> kMultiTagStdDevs = VecBuilder.fill(0.5, 0.5, 1);
   }
+
+  /**
+   * Constants for the climb subsystem
+   */
+  public static class ClimbConstants {
+    public static final int DEVICE_ID_CLIMB_MOTOR_1 = 60;
+    public static final int DEVICE_ID_CLIMB_MOTOR_2 = 65;
+    public static final int DEVICE_ID_CLIMB_ENCODER_1 = 0;
+    public static final int DEVICE_ID_CLIMB_DETECTION = 0;
+
+    // configuraton {
+    public static final Current CLIMB_STATOR_CURRENT_LIMIT = Amps.of(100);
+    public static final Current CLIMB_SUPPLY_CURRENT_LIMIT = Amps.of(40);
+    public static final double CLIMB_ROTOR_TO_SENSOR_RATIO = (1 / 25); // 25 motor/shaft turns = 1 rotor turn
```

Comment:
Sorry, this value is wrong now, it should be 25. It's 1/25th now, it was 25 before but confused me because is was 25/1 (which is 25).
Also, the comment is not correct. The rotor _is_ the motor shaft. 25 turns of the rotor is one turn of the mechanism where the sensor is mounted.